.github/workflows/ci.yaml

```diff
@@ -199,7 +199,9 @@ jobs:
                   coverage: none
             
             - name: Add solr dependency
-              run: composer require --no-update "ibexa/solr:~4.2.0@dev"
+              run: |
+                VERSION=$(cat composer.json | jq -r '.extra | ."branch-alias" | ."dev-main"')
+                composer require --no-update "ibexa/solr:$VERSION"
```

Comment:
Hmm, interesting approach indeed. Seems we won't change versioning scheme anytime soon, so looks stable. Can we have it since 4.2?

Reply:
Rebased onto 4.2